doc/conf.py

```diff
@@ -103,17 +103,18 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "python_docs_theme"
+html_theme = "furo"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {
-    "collapsiblesidebar": True,
-    "issues_url": "https://github.com/pycqa/pylint/issues/new",
-    "root_name": "PyCQA",
-    "root_url": "https://meta.pycqa.org/en/latest/",
-}
+# TO-DO: Disable thme options too see how Furo themes look in default config
+# html_theme_options = {
+#     "collapsiblesidebar": True,
+#     "issues_url": "https://github.com/pycqa/pylint/issues/new",
+#     "root_name": "PyCQA",
+#     "root_url": "https://meta.pycqa.org/en/latest/",
+# }
```

Comment:
I think we need to find a way to integrate what was in the doc previousely. Do we still have a link to github in the new style ?

Reply:
`"collapsiblesidebar": True` -> Furo by default do this
`"issues_url": "https://github.com/pycqa/pylint/issues/new"` -> By default furo will show Github and RTD link in `footer`
![image](https://user-images.githubusercontent.com/55311527/161357809-848542d7-24d7-482a-95ef-592d20523817.png)


```
"root_name": "PyCQA",
"root_url": "https://meta.pycqa.org/en/latest/",
```
these 2 options i am not familiar with, but `"https://meta.pycqa.org/en/latest/"` go to not exist page

Reply:
[furo edit button](https://pradyunsg.me/furo/customisation/edit-button/)
[furo footer](https://pradyunsg.me/furo/customisation/footer/)